packages/console/src/app/space/[did]/layout.tsx

```diff
@@ -47,9 +58,22 @@ export default function Layout ({children, params}: LayoutProps): JSX.Element {
                 {space.access?.type === 'private' ? 'Private' : 'Public'}
               </span>
             </div>
-            <label className='font-mono text-xs'>
-              {space.did()}
-            </label>
+            <div className='flex items-center gap-2 min-w-0'>
+              <label className='font-mono text-xs flex-1'>
+                <span className="md:hidden">{truncateDid(space.did())}</span>
+                <span className="hidden md:inline truncate">{space.did()}</span>
+              </label>
+              <button
+                onClick={(e) => {
+                  e.stopPropagation()
```

Comment:
suggestion: consider creating a new component for this copy button if one doesn't exist already - here's an example with better UX (ie, visual indicator that the copy succeeded): https://github.com/storacha/bluesky-backup-webapp-server/blob/main/src/components/CopyButton.tsx